content/blog/2025/06/2025-06-11-announcing-premium-tiers.md

```diff
@@ -0,0 +1,63 @@
+
+
+### What will the freemium offer look like?
+
+The idea is to set some limits for users on the free plans, which would be lifted for users on the premium plans in exchange for an affordable membership. 
+
+**We are still iterating (and will do for a while) on how it looks,** but users can expect limits around media sizes and/or volumes. The goal is to ensure that the most active users participate in covering the costs of the service, in return for the access to a fully encrypted and decentralised open network.
+
+We are limited in scope and design by the fact we need to ship a minimum viable product as soon as possible (we need to reduce costs now) and by not wanting to impose too much development (if any) to Matrix client developers.
+
+Obviously we would have preferred to keep everything free of charge. We will never sell our users’data or cripple  our services with ads, so we need to find ethical sources of revenue. 
+
+### When will the new plans take effect?
+
+The roll-out will happen progressively, starting in the coming weeks and hopefully completing in the summer of 2025. We will start by opening up premium plans to new users only, before progressively migrating all existing accounts to a free plan which will give them the option to upgrade to a premium plan. Users will of course be notified ahead of  their account being migrated.
+
+### Will this work with whatever Matrix client I use? 
+
+Yes. The plan management will be handled via the [My Account](https://account.matrix.org/account/) screens provided by the Matrix Authentication Service (MAS), and notifications to users will be sent in a dedicated room using the [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices) feature built in to the Matrix protocol – already used by the homeserver to send automatic messages to the user – so should be seamless for every client. 
+
+### I am a Matrix client developer, do I need to do anything?
+
+Assuming that your client already supports [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices), the only complication introduced by the change is for applications distributed via the Apple App Store. Apple places [restrictions](https://developer.apple.com/app-store/review/guidelines/#in-app-purchase) on how payments are implemented by iOS (et al) apps.
```

Comment:
overall that assumption will hold for most users, but in terms of spec, this is an optional feature and the spec does not require clients to implement it.

> Assuming that your client already supports [Server Notices]

Because of the above and since we also specifically point this out ("assuming"), the logical conclusion is to ask: what if not?

Reply:
```suggestion
There are two considerations from a Matrix client point of view:

- support for the [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices) feature
- if the client is distributed via the Apple App Store, then support for [MSC4286](https://github.com/matrix-org/matrix-spec-proposals/pull/4286)

If the client doesn't show server notices at all then, whilst the client will remain usable with the matrix.org homeserver, your users will have a degraded UX as they won't receive notifications when encountering usage limits.

Apple places [restrictions](https://developer.apple.com/app-store/review/guidelines/#in-app-purchase) on how payments are implemented by iOS (et al) apps that are distributed via the App Store.
```